voicevox_engine/app/routers/tts_pipeline.py

```diff
@@ -194,8 +196,8 @@ def accent_phrases(
         * カナの手前に`_`を入れるとそのカナは無声化される
         * アクセント位置を`'`で指定する。全てのアクセント句にはアクセント位置を1つ指定する必要がある。
         * アクセント句末に`？`(全角)を入れることにより疑問文の発音ができる。
-        enable_e2kが`true`のとき、テキスト中の読みが不明な英単語を、e2kで解析します。デフォルトは`true`です。
-        is_kanaが`true`のとき、enable_e2kの値は無視されます。
+        enable_katakana_englishが`true`のとき、テキスト中の読みが不明な英単語を、kanalizerで解析します。デフォルトは`true`です。
+        is_kanaが`true`のとき、enable_katakana_englishの値は無視されます。
```

Comment:
ちょっとだけ調整させていただきます！

```suggestion
        enable_katakana_englishが`true`のとき、テキスト中の読みが不明な英単語をカタカナ読みにします。デフォルトは`true`です。
        is_kanaが`true`のとき、enable_katakana_englishの値は無視されます。
```

気持ちメモ：
`kanalizer`かどうかはAPIの要件ではないので省きます！
（カタカナ読みする機能を提供していて、`kanalizer`機能を提供しているわけではないというだけ）
「解析する」だと具体的に何をするかわからないので、「カタカナ読みにする」とか「カタカナ読みに変換する」とかが良さそう！
`読みが不明な英単語`は分かりやすくて良さそう。短く書かないといけない場合は`未知の英単語`でも良さそう。もう少し漢字を減らす場合は`読みがわからない英単語`も良さそう。
`テキスト中`か`テキスト内`か迷いましたが、抽象的な範囲だと思うので「中」で良さそう！